src/components/RouteWrapper.tsx

```diff
@@ -243,7 +244,7 @@ export const ROUTES: RouteProperties[] = [
             "A walkthrough on Topology Hack the Box challenge which requires knowledge of network protocols, routing, and security configurations",
         category: "",
     },
-    //TOOLS BELOW THIS COMMENT - PLEASE ADDE NEW TOOLS IN ALPHABETICAL ORDER
+    //TOOLS BELOW THIS COMMENT - PLEASE ADD NEW TOOLS IN ALPHABETICAL ORDER
```

Comment:
Good work fixing this but be careful to stick to the task description. This would have been a good candidate for a new task.